assets/src/edit-story/components/dropzone/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { useRef, useLayoutEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useDropZone from './useDropZone';
+
+const DropZoneComponent = styled.div`
+	display: inherit;
+	position: relative;
+	${ ( { borderPosition, theme, highlightWidth } ) => borderPosition && `
+		:after {
+			height: 100%;
+			display: block;
+			position: absolute;
+			width: ${ highlightWidth }px;
+			background: ${ theme.colors.action };
+			content: '';
+			${ borderPosition }: -${ highlightWidth / 2 }px;
+		}
+	` }
+`;
+
+function DropZone( { children, onDrop } ) {
+	const dropZoneElement = useRef( null );
+	const { actions: { addDropZone, resetHoverState }, state: { hoveredDropZone, dropZones } } = useDropZone();
+
+	useLayoutEffect( () => {
+		if ( ! dropZones.some( ( { ref } ) => ref === dropZoneElement.current ) ) {
+			addDropZone( {
+				ref: dropZoneElement.current,
```

Comment:
As in the other similar case, this is a `node`, right? Not the `ref<node>`?

Reply:
Correct, changed.